examples/window-app/silabs/include/WindowManager.h

```diff
@@ -160,7 +160,8 @@ class WindowManager
 
     LEDWidget mStatusLED;
     LEDWidget mActionLED;
-
+    // Get QR Code and emulate its content using NFC tag
```

Comment:
This comment is pretty odd since we don't have a NFC

Reply:
Removed this as it is no longer needed.